mlir/include/mlir/Dialect/Vector/IR/VectorOps.td

```diff
@@ -2140,14 +2140,14 @@ def Vector_CompressStoreOp :
   Vector_Op<"compressstore">,
     Arguments<(ins Arg<AnyMemRef, "", [MemWrite]>:$base,
                Variadic<Index>:$indices,
-               VectorOfRankAndType<[1], [I1]>:$mask,
-               VectorOfRank<[1]>:$valueToStore)> {
+               VectorOf<[I1]>:$mask,
+               AnyVector:$valueToStore)> {
```

Comment:
For compress and expand, I would state in the doc that the compression/expansion applies only along the innermost dimension.

Reply:
SG